HW_2/templates/base.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>task_2</title>
+</head>
+<body>
+<form action="/submit" method="post">
```

Comment:
Тут можно убрать `action`
```suggestion
<form method="post">
```